lib/scala/filewatcher/src/test/java/org/enso/filewatcher/FileWatcherTest.java

```diff
@@ -175,8 +180,15 @@ private static void atomicAppend(Path path, String content) throws IOException {
   }
 
   private void assertNextEventIs(WatcherEvent expectedEvent) {
-    var event = pollEvent();
-    assertThat(event, is(expectedEvent));
+    LOGGER.atLevel(AT).log("Expecting event: {}", expectedEvent);
+    for (; ; ) {
+      var event = pollEvent();
+      LOGGER.atLevel(AT).log("  got event: {}" + event);
+      if (expectedEvent.equals(event)) {
```

Comment:
- There seems to be more events generated by the watcher on Mac OS X
- Thus skipping the not important ones. 
- Seems to make everything green:

![obrazek](https://github.com/user-attachments/assets/16072c3e-5241-49b3-94a1-6f66934d8d74)

- the watcher on Mac is far from optimal
- the latest attempt to improve it I can find is https://github.com/openjdk/jdk/pull/10140
- it hasn't been merged and as such people still tend to stick with native alternatives on Mac